packages/authentication/src/strategies/authentication-strategy.ts

```diff
@@ -0,0 +1,65 @@
+import {inject} from '@loopback/core';
+import {AuthenticationBindings} from '../keys';
+import {AuthenticationServices} from '../services';
+import {UserProfile, AuthenticatedUser, Credentials} from '../types';
+import {Entity} from '@loopback/repository';
+import {HttpErrors} from '@loopback/rest';
+import {toJSON} from '@loopback/testlab';
```

Comment:
I thought `@loopback/testlab` is supposed to be used in tests

Reply:
Hi @shazha thank you for catching it. This PR is still in progress, code changes everyday :( I will let you know when it's ready for review. Thanks for understanding.